requirements.txt

```diff
@@ -7,3 +7,4 @@ pycparser==2.21
 PyQt5==5.15.10
 PyQt5-Qt5==5.15.2
 PyQt5-sip==12.13.0
+ruff==0.3.5
```

Comment:
I think we don't need to require from the user to install this before run the code.

Reply:
You are free to choose whether to continue work on this issue or close it, it is your decision. If you think it's best to close it out, we can close it. Feel free to open a new issue for any features you're excited about and thanks for your work on this so far!

Reply:
Devs, present the opposing alternatives to me in a succinct, objective and technical way, so I decide on them.

Reply:
@Yuri-SVB you can find a brief description of opposing alternatives listed in the related issue #75.

Reply:
We'll continue that on issue #75.